docs/tutorials/infrastructure/binary/index.md

```diff
@@ -0,0 +1,431 @@
+
+```bash
+sudo vim /etc/systemd/system/temporal.service
+```
+
+Your unit file needs, at minimum, a [Unit] section, a [Service] section, and an [Install] section:
+
+```
+[Unit]
+Description=Temporal Service
+After=network.target
+
+[Service]
+User=temporal
+Group=temporal
+ExecStart=temporal-server -c /etc -e temporal-server start
+
+[Install]
+WantedBy=multi-user.target
+```
+
+This file can be broken down as follows:
+
+- The `[Unit]` section contains a plaintext description of your new service, as well as an `After` hook that specifies when it should be run at system startup, in this case after your server’s networking interfaces have come up.
+- The `[Service]` section specifies which command should actually be run, as well as which user should be running it. In this case, `temporal` is the user you created, and `temporal-server` is the command from the last step.
+- The `[Install]` section contains only the `WantedBy=multi-user.target` line, which works together with the `After` line in the `[Unit]` section to ensure that the service is started when the server is ready to accept user logins.
+
+Save and close the file. You can now `start` your new Temporal service, and `enable` it to run on boot automatically:
+
+```bash
+sudo systemctl start temporal
+sudo systemctl enable temporal
+```
+
+Use systemctl status temporal to verify that it started successfully. You should receive similar output to when you first ran the command in a terminal.
+
+```bash
+sudo systemctl status temporal
+```
+
+```output
+● temporal.service - Temporal Service
+     Loaded: loaded (/etc/systemd/system/temporal.service; disabled; vendo>
+     Active: active (running) since Mon 2024-07-08 11:24:40 PDT; 4s ago
+   Main PID: 19925 (temporal-server)
+      Tasks: 22 (limit: 18707)
+     Memory: 62.7M
+     CGroup: /system.slice/temporal.service
+             └─19925 temporal-server -c /etc -e temporal-server start
+
+Jul 08 11:24:42 Omelas temporal-server[19925]: {"level":"info","ts":"2024->
+Jul 08 11:24:42 Omelas temporal-server[19925]: {"level":"info","ts":"2024->
+Jul 08 11:24:42 Omelas temporal-server[19925]: {"level":"info","ts":"2024-
+```
+
+Next, repeat these steps for the UI server. Open a new file called `/etc/systemd/system/temporal-ui.service`:
+
+```bash
+sudo vim /etc/systemd/system/temporal-ui.service
+```
+
+Add the following contents:
+
+```
+[Unit]
+Description=Temporal UI Server
+After=network.target
+
+[Service]
+User=temporal
+Group=temporal
+ExecStart=temporal-ui-server -c /etc -e temporal-ui-server start
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Save and close the file, then `start` the UI Server service, and `enable` it to run on boot automatically:
+
+```bash
+sudo systemctl start temporal-ui
+sudo systemctl enable temporal-ui
+```
+
+Use systemctl status temporal to verify that it started successfully:
+
+```bash
+sudo systemctl status temporal-ui
+```
+
+Both services should now be running in the background. 
```

Comment:
I forget if this was in the actual DO style guide or not, but some folks pushed for "receive" instead of "see" on the basis that it was potentially exclusionary for folks using screen readers. I personally feel this is a pretty challenging and potentially unnecessary style point, but raising it here anyway.